events.go

```diff
@@ -111,6 +111,21 @@ information is displayed once every 5 seconds.`,
 	},
 }
 
+func convertPSI(from *cgroups.PSIData, to *cgroups.PSIData) {
+	if from == nil {
+		println("from is nil")
+		return
+	}
+	if to == nil {
+		println("to is nil")
```

Comment:
Were these leftovers from debugging this?

Reply:
yes and right now the PR is not in state of ready to review.
I think I need a decision from maintainers if it makes sense to go down the pointer change or not and if not what needs to be changed to make it able to merge.

@kolyshkin @dcantah do you know who could help me to answer these questions, because otherwise I don't feel that I want to invest my time?